src/samplers/powerspherical.jl

```diff
@@ -0,0 +1,27 @@
+# Sampler for Power Spherical
+
+struct PowerSphericalSampler{T <: Real} <: Sampleable{Multivariate,Continuous}
+    μ::Vector{T}
+    κ::T
+    dist_b::Beta
+    dist_u::HyperSphericalUniform
+end
+
```

Comment:
I think it makes sense to add a constructor like this, and then your `sampler` implementation elsewhere can just call the constructor:

```suggestion

function PowerSphericalSampler(μ, κ)
    d = length(μ)
    β = (d - 1) // 2
    dist_b = Beta(β + κ, β)
    dist_u = HyperSphericalUniform(d - 1)
    return PowerSphericalSampler(μ, κ, dist_b, dist_u)
end

```